app/src/main/java/io/github/droidkaigi/confsched2017/util/LocaleUtil.java

```diff
@@ -1,25 +1,85 @@
 package io.github.droidkaigi.confsched2017.util;
 
 import android.content.Context;
+import android.content.res.Configuration;
 import android.support.annotation.NonNull;
+import android.support.annotation.StringRes;
+import android.text.TextUtils;
 
 import java.text.DateFormat;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
+import java.util.Arrays;
 import java.util.Date;
+import java.util.List;
 import java.util.Locale;
 import java.util.TimeZone;
 
 import io.github.droidkaigi.confsched2017.BuildConfig;
+import io.github.droidkaigi.confsched2017.R;
 import io.github.droidkaigi.confsched2017.pref.DefaultPrefs;
 import timber.log.Timber;
 
 public class LocaleUtil {
 
+    private static Locale DEFAULT_LANG = Locale.ENGLISH;
+    public static final List<Locale> SUPPORT_LANG = Arrays.asList(Locale.JAPANESE, Locale.ENGLISH);
+
     private static final String TAG = LocaleUtil.class.getSimpleName();
 
     private static final TimeZone CONFERENCE_TIMEZONE = TimeZone.getTimeZone(BuildConfig.CONFERENCE_TIMEZONE);
 
+    public static void initLocale(Context context) {
+        setLocale(context, getCurrentLanguageId(context));
+    }
+
+    public static void setLocale(Context context, String languageId) {
+        Configuration config = new Configuration();
+        DefaultPrefs.get(context).putLanguageId(languageId);
+        Locale locale = new Locale(languageId);
+        Locale.setDefault(locale);
+        config.locale = locale;
```

Comment:
current PR. 
- Fixed, if/else Configuration.html#locale
- Not Fixed, `updateConfiguration has been also deprecated in API 25`